### DISCUSSION TITLE
Best practices on starting Cypress server / npm

### DISCUSSION THREAD

**Discussion by gplusdotgr** [cypress-io/cypress#21533] [Questions and Help] [answered]
Hi,

Running Cypress locally requires you to have a local web server of your APP running, (if you target localhost), are there some best practices there? For example an npm script that combines starting the server and running cypress in one command? Something I am missing?

**Comment by bahmutov** [comment, marked as answer]
- https://on.cypress.io/best-practices#Web-Servers and https://on.cypress.io/ci#Boot-your-server
